src/commands/validate.ts

```diff
@@ -41,14 +41,15 @@ export default class Validate extends SleekCommand {
     // todo check exclusive flags
     // todo check Flags.url type
     file: Flags.string({description: "Path to add-on input file", exclusive: ['helmUrl'], char:'f'}), // or file or URL, full or bits
+    directory: Flags.string({description: "Path to the addon folder", hidden: true, char:'d', exclusive: ['file', 'helmUrl']}),  // internal and for partner users who would like to test against their local code
     helmUrl: Flags.string({description: "Fully qualified URL of the Repo", exclusive: ['file']}),  // fully qualified URL of helm repo
     helmRepo: Flags.string({description: "Helm repo of the addon", exclusive: ['file', 'helmUrl'], char:'r'}),  // construct it piecemeal
     protocol: Flags.string({description: "Protocol of the helm hosting to use", exclusive: ['file', 'helmUrl'], char:'p'}),
     version: Flags.string({description: "Version of the addon to validate", exclusive: ['file'], char:'v'}),
     addonName: Flags.string({description: "Name of the addon"}),
-    skipHooks: Flags.boolean({description: "Skip helm hooks validation", default:false}),
     extended: Flags.boolean({description: "Run extended validation", hidden: true, char:'e'}), // triggers security and extended checks. NEEDS THE CONTAINER IMAGE FOR THE ADDON
-    folder: Flags.string({description: "Path to the addon folder", hidden: true, char:'d', exclusive: ['file', 'helmUrl']}),  // internal and for partner users who would like to test against their local code
+    skipHooks: Flags.boolean({description: "Skip helm hooks validation", default:false}),
+    skipResourceService: Flags.boolean({description: "Skip .Release.Service appearances", default:false}),
```

Comment:
`skipResourceService` -> `skipReleaseService` ? Also perhaps 'occurence' is a better word in this context that 'appearance' (non-native suggestion)

Reply:
fixed, tks